sympy/solvers/tests/test_ode.py

```diff
@@ -1103,11 +1110,15 @@ def test_solve_ics():
     assert solve_ics([Eq(f(x), C1*sin(x) + C2*cos(x))], [f(x)], [C1, C2],
         {f(0): 1, f(x).diff(x).subs(x, 0): 1}) == {C1: 1, C2: 1}
 
-    # XXX: Ought to be ValueError
-    raises(NotImplementedError, lambda: solve_ics([Eq(f(x), C1*sin(x) + C2*cos(x))], [f(x)], [C1, C2], {f(0): 1, f(pi): 1}))
+    assert solve_ics([Eq(f(x), C1*sin(x) + C2*cos(x))], [f(x)], [C1, C2],
+        {f(0): 1}) == {C2: 1}
+
+    K, r, f0 = symbols('K r f0')
+    eq = Eq(f(x).diff(x), r * f(x) * (1 - f(x) / K))
+    assert(dsolve(powsimp(eq), f(x), ics={f(0): f0})) == Eq(f(x), -K*f0*exp(r*x)/((K - f0)*(-f0*exp(r*x)/(K - f0) - 1)))
```

Comment:
Why are you using `powsimp` here? If that changes the form of the equation then it would be better just to change `eq` above.

Reply:
I just added the test case in #9826 as it was in the issue. I'll just change `eq` if it changes anything.

Reply:
I just added the test case in #9826 as it was in the issue. I'll just change `eq` if it changes anything.